modules/cnv-creating-vm.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-create-vms.adoc
+
+[id="cnv-creating-vm_{context}"]
+= Creating a virtual machine from the CLI
+
+The `spec` object of the VirtualMachine configuration file references
+the virtual machine settings, such as the number of cores and the amount
+of memory, the disk type, and the volumes to use.
+
+Attach the virtual machine disk to the virtual machine by referencing
+the relevant PVC `claimName` as a volume.
+
+[NOTE]
+====
+https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/[ReplicaSet]
+is not currently supported in {ProductName}.
+====
+
+See the Reference section for information about volume types and
+sample configuration files.
+
+.Domain settings
+|===
+|Setting | Description
+
+|cores
+|The number of cores inside the virtual machine. Must be a value greater than or equal to 1.
+
+|memory
+| The amount of RAM allocated to the virtual machine by the node. Specify the denomination with *M* for Megabyte or *Gi* for Gigabyte.
+
+|disks: name
+|The Name of the volume which is referenced. Must match the name of a volume.
+|===
+
+.Volume settings
+|===
+|Setting | Description
+
+|name
+|The Name of the volume. Must be a DNS_LABEL and unique within the virtual machine.
+
+|persistentVolumeClaim
+| The PVC to attach to the virtual machine. The `claimName` of the PVC must be in the same project as the virtual machine.
+|===
+
+See the
+https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachinespec[kubevirt
+API Reference] for a definitive list of virtual machine settings.
+
```

Comment:
You need to add ".Procedure" on a separate line to introduce the step.

Reply:
I combined this reference intro with the line that introduces the storage volume types ref module and reworded it a bit to include the the domain and volume settings which follow in-line. The Procedure is at the top as suggested.

The ref intro reads (again not sure I can include a module from a module)

Virtual machine storage volume types are listed here, as well as domain and volume settings. See the
https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachinespec[kubevirt
API Reference] for a definitive list of virtual machine settings.
include::modules/cnv-vm-storage-volume-types.adoc[leveloffset=+1]

Reply:
Ah, I see now this is not possible. What I will do is move the reference intro into the assembly and then call storage type table from the assembly. I think I will make modules of the domain and volume settings, too.

Reply:
Upon further thought, I am not breaking out the domain and volume settings from the CLI topic as I don't know at this point what other assemblies call in the CLI task module.

So, I have finalized on this solution, with the procedure at the top:

[id="cnv-create-vms"]
= Creating virtual machines
include::modules/cnv_document_attributes.adoc[]
:context: cnv-create-vms
toc::[]

Use one of these procedures to create a virtual machine:

* Running the virtual machine wizard
* Pasting a pre-configured YAML file with the virtual machine wizard
* Using the CLI

include::modules/cnv-creating-vm-wizard-web.adoc[leveloffset=+1]

Refer to the Virtual machine wizard fields section when running the web console wizard.

include::modules/cnv-vm-wizard-fields-web.adoc[leveloffset=+1]

include::modules/cnv-creating-vm-yaml-web.adoc[leveloffset=+1]

include::modules/cnv-creating-vm.adoc[leveloffset=+1]

Virtual machine storage volume types are listed here, as well as domain and volume settings. See the
https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachinespec[kubevirt
API Reference] for a definitive list of virtual machine settings.

Refer to the following tables when creating a virtual machine with the CLI:

* Storage volume types
* Cloud-init fields
* Networking fields
* Storage fields

include::modules/cnv-vm-storage-volume-types.adoc[leveloffset=+1]
include::modules/cnv-cloud-init-fields-web.adoc[leveloffset=+1]
include::modules/cnv-networking-wizard-fields-web.adoc[leveloffset=+1]
include::modules/cnv-storage-wizard-fields-web.adoc[leveloffset=+1]